docs/_tutorials/local_development.md

```diff
@@ -1,5 +1,124 @@
+
+## Setting up interactive components
+If you want to use features like buttons, select menus, datepickers, dialogs, or message actions, you'll need to enable interactivity.
+
+> Interactive components use the `@slack/interactive-messages` package
+
+Start by clicking **Interactive Components** on the left sidebar. Once you flip the switch, you'll see a textbox labeled **Request URL**.
+
+### What is a Request URL
+A Request URL is a public URL where Slack can send HTTP post requests.
+
+When a button is clicked, a dialog is submitted, or a user interacts with your app [using another interactive feature](https://api.slack.com/messaging/interactivity), Slack will send you information about that event, like who clicked it in what channel.
+
+### Using a local Request URL for development
+If you're just getting started with your app development, you probably don't have a publicly accessible URL yet. Eventually, you'll want to set that up, but for now a development proxy like [ngrok](https://ngrok.com/) or [localtunnel](https://localtunnel.github.io/) will do the job.
+
+Once you've installed a development proxy, run it to begin forwarding requests to a specific port (we're using port 3000 for this example):
+
+> ngrok: `ngrok http 3000`
+> localtunnel: `lt --port 3000`
+
+![Running ngrok](assets/ngrok.gif "Running ngrok")
+
+The output should show a generated URL that you can use (we recommend the one that starts with `https://`). This URL will be the base of your request URL, in this case `https://8e8ec2d7.ngrok.io`.
+
+### Add the URL to your app configuration
+Okay, so hopefully at this point you have some kind of public-facing URL. Now, let's add that to the **Interactive Components** page. Under the **Request URL** box, go ahead and paste in your accessible URL.
+
+If you're using the default HTTP server in the `@slack/interactive-messages` package, you should append `/slack/actions` to your URL. For our example, this will be `https://8e8ec2d7.ngrok.io/slack/actions`. If you are using the Express middlware, you can set whichever path you like, just remember to make the path you mount the middleware into the application the same as the one you configure in Slack.
+
+![Configuring a Request URL](assets/request-url-config.png "Configuring a Request URL")
+
+## Listening to workspace events
+If you want your app to listen to events like when a reaction is added, when a user mentions your app, or [another Events API event](https://api.slack.com/events), you'll need to enable events for your app.
+
+> The Events API uses the `@slack/events-api` package.
+
+Start by clicking **Event Subscriptions** on the left sidebar. After you toggle the switch, you'll see you need to add a Request URL. This is similar to [adding a Request URL in the interactive components section](#what-is-a-Request-URL). There are two differences though: 
+1. The default path is `/slack/events` instead of `/slack/actions`
```

Comment:
I don't like this approach because it assumes you are familiar with the setup of interactive components. "you told me i could skip around, i came from the events API readme, what the heck are interactive components?"